src/modules/Security.rb

```diff
@@ -27,6 +27,7 @@
 # $Id$
 require "yast"
 require "yast2/systemd/service"
+require "yast2/cfa/sysctl"
```

Comment:
hmm, I just see it, why not it live in `cfa/sysctl`? My idea behind living it in global cfa namespace is to make it easier in future to separate it to own gem if we decide.

Reply:
Yes, no problem. I will move it.